src/Pyz/Zed/Queue/QueueDependencyProvider.php

```diff
@@ -122,4 +123,14 @@ protected function getProcessorMessagePlugins(Container $container): array
             SearchHttpConfig::SEARCH_HTTP_CONFIG_SYNC_QUEUE => new SynchronizationStorageQueueMessageProcessorPlugin(),
         ];
     }
+
+    /**
+     * @return array<\Spryker\Zed\QueueExtension\Dependency\Plugin\QueueMessageCheckerPluginInterface>
+     */
+    protected function getQueueMessageCheckerPlugins(): array
+    {
+        return [
```

Comment:
please always use parent first in similar cases, please adjust all other integration PRs as well